Source/WebCore/Modules/modern-media-controls/controls/slider.js

```diff
@@ -130,63 +60,13 @@ class Slider extends LayoutNode
         this.needsLayout = true;
     }
 
-    get allowsRelativeScrubbing()
-    {
-        return this._allowsRelativeScrubbing;
-    }
-
-    set allowsRelativeScrubbing(allowsRelativeScrubbing)
-    {
-        this._allowsRelativeScrubbing = !!allowsRelativeScrubbing;
-
-        this.element.classList.toggle("allows-relative-scrubbing", this._allowsRelativeScrubbing);
-    }
-
     // Protected
 
-    handleEvent(event)
-    {
-        switch (event.type) {
-        case "pointerdown":
-            this._handlePointerdownEvent(event);
-            return;
-
-        case "pointermove":
-            this._handlePointermoveEvent(event);
-            return;
-
-        case "pointerup":
-            this._handlePointerupEvent(event);
-            return;
-
-        case "change":
-        case "input":
-            this._valueDidChange();
-            return;
-        }
-    }
-
-    commitProperty(propertyName)
-    {
-        switch (propertyName) {
-        case "value":
-            this._input.element.value = this._value;
-            delete this._value;
-            break;
-        case "disabled":
-            this.element.classList.toggle("disabled", this._disabled);
-            break;
-        default :
-            super.commitProperty(propertyName);
-            break;
-        }
-    }
-
     commit()
     {
         super.commit();
 
-        let scrubberWidth = (style => {
+        const scrubberWidth = (style => {
```

Comment:
Why?

Reply:
Why not? It's a value that doesn't change :)

Reply:
as a general FYI, `const` doesn't really work that way in JS :/

all it means is basically "you cannot reassign this identifier (though you can shadow it)".  it doesn't prevent modifications to the value if the value is an object (e.g. `const foo = {}; foo.a = 42;`).

i usually avoid using `const` for anything that isn't an actual constant (e.g. `42`) for this reason (as well as the fact that it's more characters to type).

NIT: my other point here was that these kinds of changes can make it more annoying for a future engineer to go back and look at the `git blame` of lines, as now they have to walk through another commit